locales/base/translation.json

```diff
@@ -75,28 +75,9 @@
   "chooseFromLibrary": "Choose from Library",
   "takePhoto": "Take Photo",
   "removePhoto": "Remove Photo",
-  "accountSendLimit": "Account Send Limit",
-  "dailyLimitLabel": "Current Daily Send Limit",
-  "dailyLimitValue": "${{dailyLimit}}",
-  "noDailyLimit": "Unlimited",
-  "verifyNumberToRaiseLimit": "You can increase your send limit by verifying your identity. Please connect your phone number before continuing.",
-  "verifyIdentityToRaiseLimit": "You can increase your send limit by verifying your identity.  Please be ready to provide some details in the following process.",
-  "dailyLimitApplicationStatus": "Application Status",
-  "applicationInReview": "In Review",
-  "applicationInReviewDescription": "Your application is being reviewed. You will be notified when this process is completed.",
-  "applicationIncomplete": "Incomplete",
-  "applicationIncompleteDescription": "The information you provided was insufficient. Please be ready to provide additional details in the following process. ",
-  "applicationDenied": "Denied",
-  "applicationDeniedDescription": "Your application was denied. If you feel this was a mistake, please restart the process below.",
   "applicationCompleted": "Completed",
   "applicationCompletedDescription": "Your application was accepted. You are now able to send an unlimited amount daily.",
-  "raiseLimitConfirmNumber": "Connect phone number",
   "raiseLimitBegin": "Begin Identity Verification",
```

Comment:
Ah is this one used in the new KYC flow?

Reply:
yeah it's used in the Persona component 😅